Header header
int32 distance_cm
